lib.php

```diff
@@ -632,6 +637,24 @@ public function event_handler($eventdata) {
             $result = true;
             if (isset($plagiarismvalues['urkund_draft_submit']) &&
                 $plagiarismvalues['urkund_draft_submit'] == PLAGIARISM_URKUND_DRAFTSUBMIT_FINAL) {
+
+                // If there's a Mahara portfolio submission, add it to the queue.
+                $submissionid = $eventdata['objectid'];
```

Comment:
This feels a bit fragile - can we check that mahara submission is enabled in this assignment first?